packages/core/src/run-engine-app.ts

```diff
@@ -78,7 +75,8 @@ export class FeatureLoadersRegistry {
      */
     async getLoadedFeatures(
         rootFeatureName: string,
-        runtimeOptions: Record<string, string | boolean> = {}
+        runtimeOptions: Record<string, string | boolean>,
+        resolvedContexts: Record<string, string>
```

Comment:
now that the change is tested, can we keep the old defaults API (rootFeatureName , ...={}, ...={}) ?